### DISCUSSION TITLE
Welcome to BIPES Discussion Forum

### DISCUSSION THREAD

**Discussion by rafaelaroca** [BIPES/BIPES#43] [Announcements]
## 👋 Welcome to BIPES Discussion Forum!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by oceanway** [comment]
I have used this program for about one month experimenting with different ESPs and have learnt a lot.
Suddenly EasyMQTT has stopped working for me.  Are other users having the same issue, or am I doing something wrong?  Thanks
